client/test/channel.go

```diff
@@ -98,7 +98,7 @@ func (ch *paymentChannel) acceptSubchannel(
 	return subchannel
 }
 
-func (ch *paymentChannel) sendUpdate(update func(*channel.State) error, desc string) {
+func (ch *paymentChannel) sendUpdate(update func(*channel.State), desc string) {
```

Comment:
```suggestion
func (ch *paymentChannel) sendUpdate(updater func(*channel.State), desc string) {
```

`updater` looks more appropriate and consistent with the previous change.